test_notebooks.py

```diff
@@ -1,21 +1,22 @@
 """ executes all Jupyter notebooks tracked by git """
 # pylint: disable=wrong-import-position
 # https://bugs.python.org/issue37373
-import sys
-
-if sys.platform == "win32" and sys.version_info[:2] >= (3, 7):
-    import asyncio
-
-    asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
-
 import gc
 import os
+import sys
 import warnings
 
 import nbformat
 import pint
 import pytest
-from git.cmd import Git
+
+from utils import find_files
+
+if sys.platform == "win32" and sys.version_info[:2] >= (3, 7):
+    import asyncio
+
+    asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
```

Comment:
@abulenok moving it here will break this workaround. We should find a way of depicting this in devops-tests CI...

Reply:
I think this is the relevant issue: https://github.com/jupyter/nbclient/issues/85

Reply:
and likely it needs to be imported before `nbformat`?

Reply:
(which is why it was placed above with the `pylint: disable=wrong-import-position` added)